### DISCUSSION TITLE
hostname_replace has been droped by PR #3552

### DISCUSSION THREAD

**Discussion by return42** [searxng/searxng#3553] [Announcements]
The old "hostname_replace" has been superseded by the [Hostnames plugin](https://docs.searxng.org/src/searx.plugins.hostnames.html).

Downward compatibility has been implemented for the transition period, which will be switched off again with the PR:

- https://github.com/searxng/searxng/pull/3552

We recommend all maintainers who are using the old “hostname_replace” to change their configuration to the new plugin as soon as possible. A description can be found here:

- https://docs.searxng.org/src/searx.plugins.hostnames.html

**Comment by ghost** [comment]
Presumably this requires `'Hostnames plugin'` to be added to `enabled_plugins` as well as the linked documentation on the new plugin?

**Comment by return42** [comment]
> Downward compatibility has been implemented for the transition period

FYI: transition period is over 

- https://github.com/searxng/searxng/pull/3552

has been merged today.

**Comment by satonotdead** [comment]
Where's the updated docs about this plugin? [Previous link](https://docs.searxng.org/src/searx.plugins.hostnames.html) seems to be broken and my configuration suddently stopped working.

**Reply by return42** [reply]
> Where's the updated docs about this plugin? [Previous link](https://docs.searxng.org/src/searx.plugins.hostnames.html) seems to be broken and my configuration suddently stopped working

Where have you found the old link ( do we need to update something?).

The docs has been moved to:

- https://docs.searxng.org/dev/plugins/hostnames.html

**Reply by satonotdead** [reply]
I got it, thanks! I found the previous link on similar issues on Github.

_You probably can redirect [the previous](https://docs.searxng.org/src/searx.plugins.hostnames.html) to the new one._

**Reply by return42** [reply]
> You probably can redirect [the previous](https://docs.searxng.org/src/searx.plugins.hostnames.html) to the new one.

Nope .. when we redirect all old links available from the WWW we have a huge mapping ;-)

So again, my question is: from where did you got the old link?

**Reply by satonotdead** [reply]
#3664 #3722
